Public/css/theme.css

```diff
@@ -14306,6 +14306,13 @@ html:not([dir=rtl]) .range-slider-ui.noUi-horizontal .noUi-handle {
   box-shadow: 0 0.375rem 1.125rem -0.375rem rgba(29, 161, 242, 0.8) !important;
 }
 
+.btn-mastodon:hover, .btn-mastodon:focus, .btn-mastodon:active {
```

Comment:
Might as well use this in the _speakers.leaf file too for speaker mastodon links